flask_resources/context.py

```diff
@@ -72,3 +73,19 @@ def inner(*args, **kwargs):
             return f(*args, **kwargs)
 
     return inner
+
+
+def with_route(f):
```

Comment:
minor: if changed name above, here to `with_route_args`.

Reply:
shelved as well.